static/shared/js/emoji.js

```diff
@@ -168,6 +168,13 @@ export function update_emojis(realm_emojis) {
                 emoji_url: data.source_url,
                 still_url: data.still_url,
             });
+        } else {
+            // this is a hack, it would be better to shift all_realm_emoji to
+            // just be based on emoji_name instead... or alternatively, to
+            // shift everywhere else from using emoji_names to emoji_codes,
+            // which is probably better in the long term, but more complex
+            // as it means we'd need to go up all the way to the data-attributes
+            deactivated_emoji_name_to_code.set(data.name, data.id);
```

Comment:
Do we need this data structure, or could we just do the lookup in `all_realm_emojis`?